doc/cephfs/client-config-ref.rst

```diff
@@ -0,0 +1,204 @@
+========================
+ Client Config Reference
+========================
+
+``client acl type``
+
+:Description: Set to ``"posix_acl"`` to enable POSIX ACLs. Modifiable via ``--inject-args``.
+:Type: String
+:Default: ``""`` (N/A)
+
+``client cache mid``
+
+:Description: Set client cache mid-point. Modifiable via ``--inject-args``.
+:Type: Float
+:Default: ``0.75``
+
+``client_cache_size``
+
+:Description: Set client cache object count maximum. Modifiable via ``--inject-args``.
+:Type: Integer
+:Default: ``16384``
+
+``client_caps_release_delay``
+
+:Description: Delay between capability releases.
+:Type: Integer
+:Default: ``5`` (seconds)
+
+``client_debug_force_sync_read``
+
+:Description: Always read synchronously (go to osds).
+:Type: Boolean
+:Default: ``false``
+
+``client_debug_getattr_caps``
+
+:Description: Check if MDS reply contains wanted caps.
+:Type: Boolean
+:Default: ``false``
+
+``client_dirsize_rbytes``
+
+:Description: Use recursive size of directory for directory ``st_size`` stat member.
+:Type: Boolean
+:Default: ``true``
+
+``client_max_inline_size``
+
+:Description: **FIXME**
```

Comment:
It's the largest a file can be and still get inlined (i.e. stored in the inode instead of in a data object).  Only applies if the inline_data flag is set in the mdsmap.